rtf_vis_tool/src/App.js

```diff
@@ -3,13 +3,13 @@ import {extend} from "react-three-fiber";
 import {OrbitControls} from 'three/examples/jsm/controls/OrbitControls';
 import './App.css';
 
-//allows the user to orbit their view around the react three fiber point cloud
-extend({OrbitControls})
+import DivGraph from './Components/DivGraph';
+extend({OrbitControls}) //allows the user to orbit their view around the react three fiber point cloud
 
 //Component which is rendered in index.js
 const App = (props) => {
   return (
-    <p>boilerplate App.js body.</p>
+    <DivGraph/>
   )
 }
 
```

Comment:
updated App.js (which was included in React PR #1) to now render DivGraph.js as the landing page